deep_qa/data/tokenizers/word_splitter.py

```diff
@@ -128,6 +139,7 @@ def split_words(self, sentence: str) -> List[str]:
 
 word_splitters = OrderedDict()  # pylint: disable=invalid-name
 word_splitters['simple'] = SimpleWordSplitter
+word_splitters['spaces'] = SpaceWordSplitter
 word_splitters['nltk'] = NltkWordSplitter
 word_splitters['spacy'] = SpacyWordSplitter
 word_splitters['no_op'] = NoOpWordSplitter
```

Comment:
I think there might be a way we could remove this completely if we expose the `TokenisedTextField` as a `Field` you are allowed to specify in a `Dataset`, but this can happen after this mega re-write.